### PR TITLE
Fix duplicate @param.

### PR DESCRIPTION
**Pull request by drupol**
This PR:

* [x] Fix the 2 current Psalm issues.

### CODE REVIEW

File: src/Writer.php [hunk 1]
```diff
@@ -58,7 +58,7 @@ public function exec(): Pair
     /**
      * @template B
      * @param callable $f
-     * @param callable(A): B $f
+     * @psalm-param callable(A):(B) $f
```
Comment (marcosh) [inline src/Writer.php:61]: there is no need to use the parentheses for the output part

**Comment by marcosh** [comment]
which version of Psalm are you using? I didn't have any issue

File: src/Writer.php (hunk 1 above)
Comment (drupol) [reply src/Writer.php:61]: Ok,I can submit a fix if you want ?

**Comment by drupol** [comment]
> which version of Psalm are you using? I didn't have any issue

^3.12.2

File: src/Writer.php (hunk 1 above)
Comment (marcosh) [reply src/Writer.php:61]: no worries, I fixed it, thanks

Comment (drupol) [reply src/Writer.php:61]: Okay!

**Pull request merged**